plugin/transfer/README.md

```diff
@@ -0,0 +1,80 @@
+# transfer
+
+## Name
+
+*transfer* - answer zone transfers requests for compatible authoritative
+plugins.
+
+## Description
+
+This plugin answers zone transfers for authoritative plugins that implement
+`transfer.Transferer`.
+
+Transfer answers AXFR requests and IXFR requests with AXFR fallback if the
+zone has changed.
+
+Notifies are not supported.
+
+*transfer* is not currently implemented by any plugins.
+
+## Syntax
+
+~~~
+transfer [ZONE...] {
+  to HOST...
+}
+~~~
+
+* **ZONES** The zones *transfer* will answer zone requests for. If left blank,
+  the zones are inherited from the enclosing server block. To answer zone
+  transfers for a given zone, there must be another plugin in the same server
+  block that serves the same zone, and implements `transfer.Transferer`.
+
+* `to ` **HOST...** The hosts *transfer* will transfer to. Use `*` to permit
+  transfers to all hosts.
+
+# Examples
+
+Answer zone transfers for `example.com.` using the theoretical plugin *plugin1*.
+
+~~~
```

Comment:
this can use `corefile` is the language, so it's tested `plugin2` must be then renamed to an existing plugin.

Reply:
I'll remove the example for now, and add it back once when there are some plugins that implement transfer exist.

I'm hoping that this version of the README, where there are no plugins that implement it transfer, will be short lived.